packages/design-tokens-schema/src/basis-tokens.test.ts

```diff
@@ -250,12 +251,12 @@ describe('common', () => {
           default: {
             'bg-document': {
               $type: 'color',
-              $value: '{ma.color.indigo.1}',
+              $value: '{brand.ma.color.indigo.1}',
```

Comment:
Should we put a `TODO` here because I think handling this is still open for debate?